new/LIT7080Qene.xml

```diff
@@ -0,0 +1,79 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7080Qene" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ወሞት፡ መንፈቀ፡ ስጋነ፡ ለረኃብ፡ ወመንፈቀ፡ ሥጋነ፡ ለመ<choice resp="JK"><sic>ሩ</sic><corr>ሬ</corr></choice>ት፤</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Wa-mot manfaqa sǝgāna la-raḫāb wa-manfaqa śǝgāna la-maret (Qǝne of the śəllāse-type)</title>
+                <title xml:lang="en" corresp="#t1">And death, half of our flesh for hunger and half of our flesh for the soil…</title>
```

Comment:
I think, the translation is Ok. I you are looking for alternatives, here is suggestion (a little more pathetic):
"As for the death, half of our flesh is for hunger, and half of our flesh is for dust"

Reply:
> Sellase is the type of six verses, and here we have three which corresponds to (za)almakiya. I have no reasonable explanation for Sellase at the beginning, I would just transcribe it as it is and translate as vague as it is.

I this case, it is wrongly attributed to Sallāśe and should be indicated as such in the abstract.

Reply:
According to Habtemichael Kidane 1998 ("L'Ufficio divino…"), p. 216, citing an Amharic source, sellase should have three verses.

Reply:
I added a note on this, but tell me if you think that this info should rather be placed in the file about "sellase-type qene" (if there is such a file)

Reply:
Yes, I agree. If there is contradictory information about what is Śəllāse type in qəne, the record should be reworked. We need to discuss, which standard should be used in Bm.

Reply:
This is a difficult case. For the moment, we stick to the system as exposed by Guidi in his "Qene o inni abissini" where he explains Sellase as a type of 6 rhymed lines. There are some other authors who might have deviations, but I suggest we better stick to this one, otherwise we will get confused. Habtemikael sometimes says things that deviate from what seems to be a common knowledge (he is from Eritrea, by the way; he is not a qene master). In EAE he says Sellase is composed of six "bets" (= three lines; vey logical as it is triple), but elsewhere I did see Sellase made of six lines, identified like this. Before Habtemikael, we have a few other authors who in my opinion are more trustful. "Sellase" at the beginning - as I told, I would leave it as vague as it is. You are not obliged to see an indication to qene-type in it. Another issue is that in such form the line does not make sense, seemingly ("Trinity and death..."???), but I had a few cases when a qene made absolutely no sense in the beginning, and this changed after a while (and many will remain untransparent).  However, I think there is no point in endless discussion here. I just approve, the rest is in your hands.